code/script.js

```diff
@@ -0,0 +1,83 @@
+// DOM selectors
+const projects = document.getElementById('projects')
+const username = 'idanaslund'
+const profile = document.getElementById('profile')
+const formRepos = document.getElementById('formRepos')
+const sortRepos = document.getElementById('sortRepos')
+
+let API_URL = `https://api.github.com/users/${username}/repos`
+
+// Fetching profile info
+const addingProfile = () => {
+    fetch(`https://api.github.com/users/${username}`)   
+    .then((res) => res.json())
+    .then((profileInfo) => {
+        profile.innerHTML += `
+        <img src="${profileInfo.avatar_url}">
+        <a class="userlink" href="${profileInfo.html_url}" target="_blank">${profileInfo.login}</a>
+        <p>${profileInfo.bio}</p>
+        `
+    }) 
+}
```

Comment:
I really like how you wrote your code here. Clean, good naming and nice that you made a link to your profile on github! :)